prisma/schema.prisma

```diff
@@ -0,0 +1,225 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+/// This table contains check constraints and requires additional setup for migrations. Visit https://pris.ly/d/check-constraints for more info.
+model addresses {
+  id              Int         @id @default(autoincrement())
+  building_number Int
+  flat_number     Int?
+  street          String      @db.VarChar(255)
+  city            String      @db.VarChar(255)
+  postcode        String      @db.VarChar(255)
+  country         String      @db.VarChar(255)
+  created_at      DateTime?   @default(now()) @db.Timestamp(6)
+  updated_at      DateTime?   @default(now()) @db.Timestamp(6)
+  customers       customers[]
+}
+
+/// This table contains check constraints and requires additional setup for migrations. Visit https://pris.ly/d/check-constraints for more info.
```

Comment:
To ci robil chat, czy jakis generator, bo nie widzialem nigdy tego a wydaje sie fajne

Reply:
do tego jest komenda  _prisma db pull_ albo cos takiego